lib/spack/llnl/util/cpu/microarchitectures.json

```diff
@@ -705,13 +738,14 @@
         "abm",
         "avx",
         "xop",
-        "lwp",
+        "fma4",
         "aes",
         "pclmulqdq",
         "cx16",
         "ssse3",
         "sse4_1",
-        "sse4_2"
+        "sse4_2",
+        "popcnt"
```

Comment:
@chissg I see `popcnt` isn't in the gcc list of flags it checks for bulldozer. Is there a reason to add it here?

Reply:
I included it as part of the bulldozer flags for consistency with the k10 list. Other than that, no reason. I will remove it for consistency with the other family 15h lists.